core/modules/loader/appClientNum.cc

```diff
@@ -151,17 +151,24 @@ int main(int argc, char* argv[]) {
     cClient.run();
     cClient.run();
     cClient.run();
+    cClient.run();
 
     KeyInfoDataList kList;
     int successCount = 0;
     int failedCount = 0;
+    int totalKeyCount = 0;
+
+    TimeOut::TimePoint insertBegin = TimeOut::Clock::now();
+
     if (numEnd >= numStart) {
+        totalKeyCount = (numEnd - numStart) + 1;
         for (uint64_t j=numStart; j<=numEnd; ++j) {
             kList.push_back(clientAdd(cClient, j));
             // occasionally trim the list
             if (j%1000 == 0) keyInsertListClean(kList, successCount, failedCount);
```

Comment:
What's so special about `1000`? Why not `100` instead?
It may make a sense to do the trimming based on something like an interval of time or the total number of requests in a queue, or on the memory, etc.